tokio/src/io/registration.rs

```diff
@@ -296,24 +300,26 @@ impl Registration {
         // AND write. A specific case that `EPOLLERR` occurs is when the read
         // end of a pipe is closed. When this occurs, a peer blocked by
         // writing to the pipe should be notified.
-        let curr_ready = sched
-            .set_readiness(self.address, |curr| curr & (!mask_no_hup))
-            .unwrap_or_else(|_| panic!("address {:?} no longer valid!", self.address));
```

Comment:
nit: it looks like the previous panic message was a bit more helpful..

Reply:
The call to `set_readiness` now can't hit a code path that returns `Err`. The only panic message that makes sense is "Tokio hit a bug, plz report" :)

Reply:
I can panic w/ `unreachable!()` instead...